cmd/cogito/main.go

```diff
@@ -37,16 +36,27 @@ func mainErr(in io.Reader, out io.Writer, logOut io.Writer, args []string) error
 	if err != nil {
 		return fmt.Errorf("reading stdin: %s", err)
 	}
+
 	logLevel, err := peekLogLevel(input)
 	if err != nil {
 		return err
 	}
-	log := hclog.New(&hclog.LoggerOptions{
-		Name:        "cogito",
-		Level:       hclog.LevelFromString(logLevel),
-		Output:      logOut,
-		DisableTime: true,
-	})
+	var level slog.Level
+	if err := level.UnmarshalText([]byte(logLevel)); err != nil {
+		return fmt.Errorf("%s. (valid: DEBUG, INFO, WARN, ERROR)", err)
+	}
+	removeTime := func(groups []string, a slog.Attr) slog.Attr {
```

Comment:
Just for my understanding: why you don't want the timestamp?

Reply:
We are disabling timestamps in two places: in prod code (here) and in the tests.

In the tests, because timestamps make tests more difficult, when doing string matching.

In prod code (here), also with hclog we were disabling timestamps, because they are already provided by the Concourse UI. See the removed line 48 just above.

I don't have a strong opinion, if we think it would help adding them in prod (this would allow for example to have them when doing a mouse copy and paste)...

Reply:
Ok. Lets keep the same behavior. It will be simple to add later the timestamp if we see the need. 
Thanks for the explanation.